docs/Contributing.asciidoc

```diff
@@ -267,6 +267,17 @@ We use annotations in some places to mark "uncoverable" code such as this:
 
 See the docs for details https://metacpan.org/pod/Devel::Cover
 
+To run the tests with a different command than `prove` you can set the
+`PROVE` variable in the Makefile. For example, in our CircleCI tests we
+use `tools/prove_wrapper` to run the tests. This wrapper is a simple script that
+will run `prove` and fail the test if there is unhandled output. This is useful to
+avoid having unhandled output in tests. To run the tests with `tools/prove_wrapper` you can do:
+
+[source,sh]
+----
+timeout -s SIGINT -k 5 -v $((25 * 2 * (0 + 1) ))m tools/prove_wrapper -l --timer t/foo.t
```

Comment:
this looks a bit weird to me. why can't we use `make test` here?

Reply:
It's the fastest way to test the unhandled output, but to proper emulate the circleci and use make we can use:

```
RETRY=0 TIMEOUT_M=25 PROVE_ARGS='--timer t/foo.t' PROVE=tools/prove_wrapper make test-unit-and-integration
```

Reply:
without the `--timer` is failing due to `set -e` suggested to add here https://github.com/os-autoinst/openQA/pull/6382#discussion_r2054335351

Reply:
It's failing due to the grep that define `UNHANDLED` variable, I can suppress by adding a false positive `|| true` after the `grep`.

Reply:
I've update the script with `UNHANDLED=$(grep -vE '^t/.*([0-9]{2}-|)([[:alpha:]]|_|-)+\.t \.+' "$OUTPUT.processed" || true)` and the documentation accordingly `PROVE=tools/prove_wrapper make test TESTS=t/foo.t`